tests/config/testConfigs/webdav/token

```diff
@@ -0,0 +1 @@
+XXXXXXXX
```

Comment:
missing new line? Or is a newline not going to work?

Reply:
Will work even with a newline, I trim those when setting the header. Added one here.